io.itch.amcsquad.amcsquad.yml

```diff
@@ -0,0 +1,104 @@
+              timestamp-query: first | .commit.committed_date
+              version-query: $tag | sub(\"^[vV]\"; \"\")
+              is-main-source: true
+              url-template: https://gitlab.com/fpiesche/amc-squad-data/-/archive/v$version/amc-squad-data-v$version.tar.bz2
+        build-commands:
+          - mkdir -p /app/bin
+          - cp -rv ./* /app/bin/
+    sources:
+      - type: archive
+        url: https://github.com/dibollinger/amcduke32/archive/refs/tags/4.1.0.zip
+        sha256: "d7631acfc1a63980b4297ebf3da30acaf8b2e8b5d19e832560d27b2418a17da9"
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/dibollinger/amcduke32/tags
+          version-query: .[0].name
+          url-query: .[0].zipball_url
+      # Silence compiler warning spam
+      - type: patch
+        path: amcduke32_01_basic_fixes.patch
+      # Add new filesystem functionality used by further patches
+      - type: patch
+        path: amcduke32_02_new_filesystem_functions.patch
+      # Redirect texture cache to XDG_DATA_HOME
+      - type: patch
+        path: amcduke32_03_redirect_texture_cache.patch
+      # Redirect GRP file list caceh to XDG_DATA_HOME
+      - type: patch
+        path: amcduke32_04_redirect_grpcache.patch
+      # Redirect configuration files to XDG_DATA_HOME
+      - type: patch
+        path: amcduke32_05_redirect_config_files.patch
+      # Redirect log files to XDG_DATA_HOME
+      - type: patch
+        path: amcduke32_06_redirect_logs.patch
+      # Redirect menus for selecting user maps and soundfonts to XDG_DATA_HOME
+      - type: patch
+        path: amcduke32_07_redirect_menus.patch
+      # Redirect read/write from game scripts to XDG_DATA_HOME
+      - type: patch
+        path: amcduke32_08_redirect_gamescript_rw.patch
+      # Redirect demo recording to XDG_DATA_HOME
+      - type: patch
+        path: amcduke32_09_redirect_demos.patch
+      # Redirect screenshots to XDG_DATA_HOME
+      - type: patch
+        path: amcduke32_10_redirect_screenshots.patch
+      # Redirect savegames to XDG_DATA_HOME
+      - type: patch
+        path: amcduke32_11_redirect_savegames.patch
+      # launcher script
+      - type: script
+        dest-filename: io.itch.amcsquad.amcsquad.sh
+        commands:
+          - cd /app/bin
+          - ./amcsquad
+      # FreeDesktop support files
+      - type: file
+        path: io.itch.amcsquad.amcsquad.desktop
+      - type: file
+        path: io.itch.amcsquad.amcsquad.png
+      - type: file
+        path: io.itch.amcsquad.amcsquad.metainfo.xml
+    buildsystem: simple
+    builddir: true
+    build-options:
+      cxxflags: "-std=c++20"
+    build-commands:
+      - make -j$FLATPAK_BUILDER_N_JOBS
+      - install -Dm 744 amcsquad -t /app/bin
+      - install -Dm 744 io.itch.amcsquad.amcsquad.sh -t /app/bin
+      - desktop-file-edit --set-key=Exec --set-value=io.itch.amcsquad.amcsquad.sh io.itch.amcsquad.amcsquad.desktop
```

Comment:
why not making this right from the get go since it's included here?

Reply:
These are the same files I had submitted upstream, so the desktop file was still using the regular executable name. Removed that for now and will reinstate it once my upstream PR gets merged.